bayesian/api/user_api.py

```diff
@@ -40,17 +42,20 @@ def generate_uuid_for_user():
 def create_or_update_user():
     """Endpoint for creating or updating user details."""
     content = request.json
```

Comment:
can we check for empty payload here..?

Reply:
It is being checked.  The content is null condition takes care of it.

Reply:
Sorry, I missed it.